scripts/fiber/tests-passing-except-dev.txt

```diff
@@ -51,14 +51,25 @@ src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js
 * renders a div with text with flanking whitespace with client render on top of bad server markup
 * renders a div with text with client render on top of bad server markup
 * renders a div with blank text child with client render on top of bad server markup
+* renders renders a div with blank text children with client render on top of bad server markup
```

Comment:
Good catch! I fixed that in 091d7fd and made some of the other names more descriptive.